[website] Clarify scope of technical support (#33435)
mui/material-ui: docs/data/material/getting-started/support/support.md

Find details on the [supported versions](/versions/#supported-versions).

> ## Technical support (premium)

Author: How should we name this one?
@mbrookes @oliviertassinari 

```suggestion
## Professional technical support
```

Reviewer: > it's understandable that some users create a connection with MUI Core.

I think that it depends on the scope we want for this page, so far, it's not meant to be about MUI Core, but MUI.

Find details on the [supported versions](/versions/#supported-versions).

## Paid support

### Pro/Premium plans

MUI X comes in [three plans](https://mui.com/pricing/).

> The support available on the Community plan is made possible thanks to people like you: the community.

Reviewer 2: ```suggestion
The support available under the Community plan is made possible thanks to people like you: the community.
```

> It's described in more detail in the section above.

Reviewer 2: ```suggestion
This is described in more detail in the section above.
```

MUI X maintainers focus on solving root issues rather than offering direct support to the community at large.

> The Pro plan offers developers advanced components and extra features that are challenging to find in OSS.

Reviewer 2: ```suggestion
The paid plans offer developers advanced components and extra features that are challenging to find in OSS.
```

> To provide a similar quality of experience to the one provided by the community on the MIT modules,

Reviewer 2: ```suggestion
To provide a similar quality of experience to the one provided by the community for MIT licensed code,
```

> the maintainers of MUI provide support for the advanced components included in MUI X.

Reviewer 2: ```suggestion
the developers of MUI  X provide support for the advanced components only.
```

> This plan includes direct technical support from MUI X maintainers to MUI X components exclusively.\

Reviewer 2: ```suggestion
```

Technical support for MUI Core components is **not** included.

The Premium plan provides developers with the highest priority for support tickets.

> No SLAs are provided yet, it's a work in progress.

Reviewer 2: ```suggestion
No SLAs are provided yet; it's coming.
```

> Please visit [MUI X Support page](https://mui.com/x/introduction/support/#technical-support) for more detailed information on technical support.

Reviewer 2: ```suggestion
Please visit [the MUI X Support page](https://mui.com/x/introduction/support/#technical-support) for more detailed information on technical support.
```

MUI X comes in [three plans](https://mui.com/pricing/).

> The support available on the Community plan is mostly made possible thanks to people like you: the community. It's described in more detail in the above section. The maintainers of MUI focuses on solving the root issues, not direct support.

Reviewer 3: ```suggestion
The support available on the Community plan is made possible thanks to people like you: the community. 
It's described in more detail in the section above. 
MUI X maintainers focus on solving root issues rather than offering direct support to the community at large.
If you need support from us, you'll need to purchase one of the commercial licenses described below.
```

Author: "If you need support from us, you'll need to purchase one of the commercial licenses described below."

This is something we need to clarify, that by buying the commercial licenses, you don't get support on the Core components.

```suggestion
The support available on the Community plan is made possible thanks to people like you: the community.
It's described in more detail in the section above. 
MUI X maintainers focus on solving root issues rather than offering direct support to the community at large.
```

The Pro plan allows developers to license components and extra features that are challenging to find in the OSS space.

> To provide a similar quality of experience to the one provided by the community on the MUI Core modules,

Reviewer: MUI X is also part of the equation:

```suggestion
To provide a similar quality of experience to the one provided by the community on the MIT modules,
```

> the maintainers of MUI provide support for the advanced components included in MUI X.

Reviewer: We could, but I don't think that the reason mentioned is correct. It's not because MUI X operates independently from the Core team that we don't do it. I think that we don't do it because it's not what we want to sell in this plan, I suspect it would require a different pricing structure.

> To provide a similar quality of experience to the one provided by the community on the MUI Core modules,
> the maintainers of MUI provide support for the advanced components included in MUI X.

Author: ```suggestion
To provide a similar quality of experience to the one provided by the community on the MUI Core modules,
the maintainers of MUI provide support for the advanced components included in MUI X.

This plan includes direct technical support from MUI X maintainers to MUI X components exclusively. 
Technical support for MUI Core components is not included.
```

> the maintainers of MUI provide support for the advanced components included in MUI X.

Author: Suggestion applied.

> The Premium plan goes one step further and gets the highest priorities on tickets. No SLAs are provided.

Reviewer 3: Trimming the fat.

```suggestion
The Premium plan provides developers with the highest priority for support tickets. 
No SLAs are provided.
```

> Please visit [MUI X Support page](https://mui.com/x/introduction/support/#professional-support) for more detailed information on professional support.

Reviewer: ```suggestion
Please visit [MUI X Support page](https://mui.com/x/introduction/support/#technical-support) for more detailed information on technical support.
```
